drivers/clock_control/clock_control_nrf.c

```diff
@@ -620,6 +620,26 @@ static void clock_event_handler(nrfx_clock_evt_type_t event)
 	}
 }
 
+static void hfclkaudio_init(void)
+{
+#if DT_NODE_HAS_PROP(DT_NODELABEL(clock), hfclkaudio_frequency)
+	const uint32_t frequency =
+		DT_PROP(DT_NODELABEL(clock), hfclkaudio_frequency);
+	/* As specified in the nRF5340 PS:
+	 *
+	 * FREQ_VALUE = 2^16 * ((12 * f_out / 32M) - 4)
+	 */
+	const uint32_t freq_value =
```

Comment:
ditto

Reply:
These two variables are only used for preparation of the parameter for `nrf_clock_hfclkaudio_config_set()`, they don't need to be stored in any way, so keeping them as automatic just feels more natural. Actually, in both cases (`static` or not) they are supposed to be optimized out anyway, so it does not really matter (and the current way saves one word in each of those lines :wink:).